lib/atat-iam-stack.ts

```diff
@@ -116,6 +116,7 @@ export class AtatIamStack extends cdk.Stack {
     });
 
     const baseDenies = new iam.ManagedPolicy(this, "AtatUserDenyPolicy", {
+      description: "Denies access to all resources not explicitly granted in other IAM policies",
```

Comment:
That's not really what this does. This would actually be interesting to turn into a permissions boundary...

What this is actually doing is just denying access to Organizations _explicitly_ just in case we give someone `*` (which we do). We really don't want folks mucking with Organizations.